tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -802,6 +802,45 @@ def _check_agent_log_on_test_nodes(self, ignore_error_rules: List[Dict[str, Any]
                     "Error checking agent log",
                     add_exception_stack_trace=True)
 
+            try:
+                message = f"Checking that there were no issues with the log collector on test node {node_name}, starting at {check_log_start_time.strftime('%Y-%m-%dT%H:%M:%S.%fZ')}"
```

Comment:
what are we checking for log collector? are we checking for every suite and every distro? will this add more run time to complete all test and also looks overhead for adding every test

The other thing is log collector will start after 5 mins of agent start? if test_suite completes before that this never validates log collector run.

you may want to change that 5min delay to debug flag and change according to our test and we disable the code for v2. This way it misses out testing log collector run. 

So you may think about adding new test and change the flags internally to that test